python/plugins/processing/algs/qgis/IdwInterpolation.py

```diff
@@ -29,28 +29,28 @@
 
 from qgis.PyQt.QtGui import QIcon
 
-from qgis.core import QgsRectangle, QgsWkbTypes
+from qgis.core import QgsRectangle
 from qgis.analysis import (QgsInterpolator,
                            QgsIDWInterpolator,
                            QgsGridFileWriter
-                           )
+                          )
 
 from processing.core.GeoAlgorithm import GeoAlgorithm
 from processing.core.GeoAlgorithmExecutionException import GeoAlgorithmExecutionException
-from processing.core.parameters import ParameterVector
-from processing.core.parameters import ParameterSelection
-from processing.core.parameters import ParameterNumber
-from processing.core.parameters import ParameterExtent
+from processing.core.parameters import (Parameter,
+                                        ParameterNumber,
+                                        ParameterExtent,
+                                        _splitParameterOptions
```

Comment:
This is helper function from parameters module, and it uses only inside this module. I will look if we can remove prefix or avoid using it here